DrinkManagerConsole/SearchDrinkConsoleUi.cs

```diff
@@ -332,7 +341,72 @@ public static void WriteDrinkInfo(Drink drink)
             Console.WriteLine($"\nInstructions:\n{drink.Instructions}");
             Console.WriteLine(
                     "------------------------------------------------------------------------------------------------------------------");
-            return;
+            if (drink.isReviewed)
+            {
+                Console.WriteLine($"Score: {drink.DrinkReview.ReviewScore}");
+                Console.WriteLine(drink.DrinkReview.ReviewText);
+                Console.WriteLine($"Date: {drink.DrinkReview.ReviewDate}");
+                Console.WriteLine(
+                    "------------------------------------------------------------------------------------------------------------------");
+            }
         }
+
+        public static void DisplayProductDetailsOptions(Drink drink)
+        {
+            while (true)
+            {
+                Console.WriteLine();
+                if (!drink.isReviewed)
+                {
+                    Console.WriteLine("Press R to leave a review for this drink");
+                }
+                else if (ValidateReviewAge(drink.DrinkReview.ReviewDate))
+                {
+                    Console.WriteLine("Press R to edit review for this drink");
+                }
+                Console.WriteLine("Press F to add to favorites list");
+                Console.WriteLine("Press E to edit drink details");
+                Console.WriteLine("\nPress ESC to go back");
+
+                var userChoice = Console.ReadKey(true);
+
+                switch (userChoice.Key)
+                {
+                    case ConsoleKey.R:
+                        if (!drink.isReviewed)
+                        {
+                            // Create review method call
+                            break;
+                        }
+                        else if (ValidateReviewAge(drink.DrinkReview.ReviewDate))
+                        {
+                            // Edit review method call
+                            break;
+                        }
+                        else
+                        {
+                            Console.WriteLine("\nUnsupported input, try again...\n");
+                            continue;
+                        }
+
+                    case ConsoleKey.F:
+                        // Add to favorites method call
+                        break;
+
+                    case ConsoleKey.E:
+                        // Edit drink details method call
+                        break;
+
+                    case ConsoleKey.Escape:
+                        return;
+
+                    default:
+                        Console.WriteLine("\nUnsupported input, try again...\n");
+                        continue;
+                }
+            }
+        }
+
+        private static bool ValidateReviewAge(DateTime reviewDate) => !(DateTime.Now.Subtract(reviewDate).Seconds > 5);
```

Comment:
Property Seconds returns the second component of the time interval and return value ranges from -59 to 59, which does not address review age in seconds. Probably better solution here would be TotalSeconds, which gets the value of the time interval expressed in seconds. Condition > 60.